index.html

```diff
@@ -80,5 +80,164 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
+            <action id="action">
+              <img src="./assets/Mask Group.png" alt="Professional art data">
+            </action>
+            <a href="https:github.com"><div id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </div></a>
+
+            <li id="card_data">
+              <card_title id="card_title"><p>Profesional Art Printing Data</p></card_title>
+              <card_secondary id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></card_secondary>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+
+              </card_tags>
+            </li>
+          </div>
+
+          <div id="card_works">
+            <action id="action">
+              <img src="./assets/Mask Group.png" alt="Professional art data">
+            </action>
+            <a href="https:github.com"><div id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </div></a>
+
+            <li id="card_data">
+              <card_title id="card_title"><p>Profesional Art Printing Data</p></card_title>
+              <card_secondary id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></card_secondary>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+
+              </card_tags>
+            </li>
+          </div>
+        </ul>
+      </div>
+
+    <div id="about_me">
+      <me_primary_text><p id="me_primary_text">About me</p></me_primary_text>
+      <me_secondary_text><p id="me_secondary_text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></me_secondary_text>
+      <a href="https://github.com/glitcher255/micro-porfolio"><me_btn class="orange_btns" id="me_btn"><p id="me_btn_text">Get My Resume</p></me_btn></a>
+      <me_art id="me_art"></me_art>
+      <me_rectangle id="me_rectangle"><img src="./assets/Rectangle 73.png" alt="Rectangle"></me_rectangle>>
+      <me_vector id="me_vector"><img src="./assets/Vector 4.png" alt="Divider"></me_vector>
+    </div>
```

Comment:
- [ ] Kindly put back the section about the skills, it is part of the requirements as you can see they put it for template 1 but it applies to the other templates too. Please use the structure mentioned by the previous reviewer 👍 

## Project requirements:
![image](https://user-images.githubusercontent.com/50721479/205439223-7668b9a3-95e3-433e-8dae-03a3b90e4430.png)
![image](https://user-images.githubusercontent.com/50721479/205439165-53c3d688-8a01-4f79-b844-a4bdc3552704.png)

Reply:
Finished.